source/mainnet/net/guides/export-import.rst

```diff
@@ -5,20 +5,50 @@
 Make a backup of identities, accounts, and addresses
 ====================================================
 
-To make sure that you have a backup of your accounts, identities, and addresses, Concordium strongly recommends that you export the data to a file you can store in a safe location. The backup will ensure that you can recover your accounts, identities, and addresses if your Wallet database becomes damaged or if, for some reason, you can't access the Wallet.
-
 .. Note::
 
-    Backup and import cannot be used for |mw-gen2| because it uses a secret recovery phrase to recover the wallet. For more information, see :ref:`Recover your wallet<recover-wallet>`.
+   This information is not relevant for |mw-gen2| or |bw| which use a secret recovery phrase to recover a wallet. For more information, see :ref:`Recover your wallet<recover-wallet>`.
 
-.. Warning::
-    You can't import a file created in the |mw-gen1| into the Desktop Wallet or the other way around because the two wallets handle private keys in different ways. You also cannot import a backup from the |mw-gen1| to the |mw-gen2|. If you try to import a file that has been exported from the |mw-gen1| into the Desktop Wallet, the import will fail, and likewise, if you try to import a file exported from the Desktop Wallet into the |mw-gen1|. For more information, see :ref:`Deciding between the Wallets <choosing-wallet>`.
+To make sure that you have a backup of your accounts, identities, and addresses, Concordium strongly recommends that you export the data to a file you can store in a safe location. The backup will ensure that you can recover your accounts, identities, and addresses if your Wallet database becomes damaged or if, for some reason, you can't access the Wallet.
 
 .. Warning::
    You are solely responsible for keeping your assets secure. You must never share your private keys, PIN codes, passwords, recovery phrases, LEDGER devices, or mobile devices with anyone.
 
+A backup is only necessary when creating new accounts, not every time a transaction is executed. Think of your account like a safe: it contains assets. If you lose the key, you cannot get into the safe unless you have a copy of the key. Your backup is your copy of the safe key.
+
+You can make a backup from |mw-gen1| or from Desktop Wallet but there are differences between them.
```

Comment:
I see, this comes at the end. I would recommend rewriting this section a bit to make it clear 
- if you use legacy wallet or desktop -> make a backup file
- if you use other wallets -> secure you seedphrase and make sure you have a backup of that.

Reply:
There is also a note at the top of the topic that states that this isn't relevant for users of the new android wallet or browser wallet and that they should see the recover wallet topic.